content/blog/getting-started-with-the-hpe-greenlake-developer-portal.md

```diff
@@ -0,0 +1,88 @@
+
+Each service section offers, at a minimum, an overview, a guide, the Open API specification, and the associated API reference documentation. Some services provide additional parts to dive into more specific's areas of the service. For instance, HPE GreenLake for Compute Ops Management service provide additional details about event webhooks and jobs. A webhook is an HTTP-based callback function that allows lightweight, event-driven communication between two APIs.  A job is a multi-step task that is managed by Compute Ops Management to perform an action on a resource. For example, performing power operations or firmware updates on a server. 
+
+The Overview gives a definition of the service. The Guide explains the core details of the service from an API standpoint. Finally, the API reference documentation will provide you with a download link to the Open API specification file in JSON format of the API. This JSON file can be used in Postman, allowing you to build collection of REST API calls. In fact, the HPE Developer Community team has already put together a nice HPE GreenLake platform API collection. You can get the Postman collection from the [HPE Developer Community tooling GitHub repository](https://github.com/hpe-dev-incubator/GLP-API-Tooling/tree/main/Postman-Collections)
+
+Finally, “la cerise sur le gateau” as we say in French, the try part. There you can make real API calls related to the selected service, bearing in mind that you have an HPE account, joined a workspace, and that you have generated the necessary API client credentials and access token. 
+
+By simply clicking the Try it button, you can get access to the API. 
+
+![](/img/blog-greenlake-dev-portal3.png)
+
+Now, you can pass your first API call. Simply paste the token in the security window and hit the Send button. 
+
+![](/img/blog-greenlake-dev-portal4.png)
+
+As a result, a 200 status will inform you that the call was passed correctly, and the Headers window will display the result of the call. 
+
+On the right-hand side of the windows, you will see that samples are being presented. You will find Curl, JavaScript, Node.js, Java, Python versions that you can copy / paste to test, allowing you to integrate them easily. 
+
+![](/img/blog-greenlake-dev-portal5.png)
+
+In addition, a Change Log section will inform you about the versioning of the API and changes related to it.
+
+## HPE Developer Community
+
+T﻿her third tile allows you to reach out to us,  \[the HPE Developer Community](https://developer.hpe.com/). Our web portal as mentionned in the introductionnary words of this blog, will help you find all things software at HPE.[Join us](https://developer.hpe.com/community) to collaborate and build applications and integrations with HPE products using the latest software and open source technologies.
```

Comment:
There's a typo with the first word here. It should say "The". I don't think you have the proper notation around the HPE Developer Community. See how it shows in the preview:

![image](https://github.com/hpe-dev-incubator/hpe-dev-portal/assets/77496261/e8a184af-207b-4cdf-88ce-1f8d7a88fa4b)

There are also a few typos here. There should be a comma after "Our web portal". There's only one "n" before "ed" in "mentioned". I would change "in the introductionnary words of this blog" to "...as mentioned in the introduction above, will help you find..."

And then you're missing a space between "HPE" and "Join us"